docs/calico-with-docker/docker-network-plugin/README.md

```diff
@@ -93,7 +93,7 @@ containers is not required.
 
 The Calico IPAM driver assigns addresses with host aggregation - this is a more
 efficient approach for Calico requiring fewer programmed routes.  IPv6
-addresses are supported, although due limitations with Docker, it is not 
+addresses are supported, although due to limitations with Docker, it is not
```

Comment:
Not you I realise - but could you tweak this to "although with the current Docker API"